include/aie/Dialect/AIE/IR/AIETargetModel.h

```diff
@@ -60,11 +60,11 @@ class AIETargetModel {
     TK_AIE1_Last,
     TK_AIE2_VE2302,
     TK_AIE2_VE2802,
-    TK_AIE2_NPU1,
-    TK_AIE2_NPU1_1Col,
+    TK_AIE2_NPU1_First,
+    TK_AIE2_NPU1_1Col = TK_AIE2_NPU1_First,
     TK_AIE2_NPU1_2Col,
     TK_AIE2_NPU1_3Col,
-    TK_AIE2_NPU1_4Col,
+    TK_AIE2_NPU1, // whole array must be last because of how we cast/initialize the VirtualizedNPU1TargetModel class
```

Comment:
This doesn't seem right. `TK_AIE2_NPU1` is mentioned here but deleted everywhere else. It shouldn't be needed at all if `NPU1TargetModel` is deleted. `TK_AIE2_NPU1_4Col` shouldn't need a rename.